src/snowflake/connector/cursor.py

```diff
@@ -1106,11 +1121,18 @@ def wait_until_ready():
         self._sfqid = sfqid
         self._prefetch_hook = wait_until_ready
 
+    def get_result_batches(self) -> Optional[List["ResultBatch"]]:
```

Comment:
This is the main exposed interface, let us have some comments here.

Reply:
Oops, good catch